R/drive_list.R

```diff
@@ -0,0 +1,162 @@
+#' List files on Google Drive
+#'
+#' @param path character vector, path name(s) where the Google drive files are
+#'   that you would like to list. Defaults to the "My Drive" directory.
+#' @param pattern character vector, regular expression(s) of title(s) of
+#'   documents to output in a tibble. If it is `NULL` (default), information
+#'   about all documents in drive will be output in a tibble.
```

Comment:
I would eliminate the last sentence, because many other arguments might be filtering the results.

I'm going to make a high-level comment on this elsewhere but given the power of `q`, what is the point of `pattern`? All other things being equal, it makes sense to implement all filters via `q` to make your package snappier. But I think `pattern` is required to filter names on a regular expression, right? That's why we implement this *post hoc* on the R side.